.github/workflows/main.yml

```diff
@@ -70,7 +70,7 @@ jobs:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
         toolchain: [stable]
-        mode: ['', '--release']
```

Comment:
This expands the test matrix by 50%.

Let's treat the p256k1 as a "tier 2" configuration, and only test if it compiles and runs tests on a single Linux runner? I presume the implementation is pure Rust, so we don't get much by testing it on other OSes.